lib/gluegun.rb

```diff
@@ -88,14 +90,12 @@ def self.gluegun_generate_pages(site_config_file)
             end
           end
           # Calling an empty puts to create a new line.
```

Comment:
Very minor thing here, I don't think we need this comment anymore as we aren't printing an empty line here